.github/workflows/coverage-gh-pages.yml

```diff
@@ -41,6 +41,8 @@ jobs:
         run: ninja -C build check-all
       - name: Generate Report
         run: ninja -C build generate-coverage-report
+      - name: Force artificat permissions
+        run: chmod -c -R +rX ${{github.workspace}}/build/report
```

Comment:
Is the -c for diagnostics? I don't know how many files are typically in this directory for how chatty that might be.

Reply:
It only reports when a change is made to the permissions. I'd like to keep that in to see how wide the impact is, but will silence in a follow-up if the log is too big.